variables.tf

```diff
@@ -3,6 +3,18 @@ variable "name" {
   type        = string
 }
 
+variable "app_version" {
+  description = "Version of rdb loader redshift"
+  type        = string
+  default     = "5.8.0"
```

Comment:
Same query as for transformer - does this version still work with the current config baked into this module?

Reply:
thanks for the Q, yes it should work

Reply:
pushed the release commit!